xml/docu_styleguide.structure.xml

```diff
@@ -2667,10 +2667,79 @@ xml:id="tab-install-source"</screen>
-   list.
-  </para>
+
+  <para>Typical use cases of tables include:</para>
+  
+  <variablelist>
+   <varlistentry><term>Lookup tables for specific information</term>
+    <listitem>
+     <para>Whenever users need to check for data that applies to them, 
+      create lookup tables. For example, use a table to sum up system requirements.
+      </para>
+    </listitem>
+   </varlistentry>
+   <varlistentry><term>Matrix tables</term>
+    <listitem>
+     <para>Whenever users need to quickly check whether a specific combination
+      of options works or not. For example, use a matrix table to visualize supported 
+      combinations or update options.</para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+
+  <para>As there are use cases for tables, there are cases when <emphasis>not</emphasis> 
+   to use tables:</para>
+  
+  <itemizedlist>
+   <listitem>
+    <para>Value and description pairs are better handled by means of a descriptive list.</para>
+   </listitem>
+   <listitem>
+    <para>Wordy explanations or descriptions should not be used in tables.</para>
+   </listitem>
+   <listitem>
+    <para>Complex layout constructs (screens, code) should not be used in tables.</para>
+   </listitem>
+  </itemizedlist>
+
+  <para>Some style tips for tables:</para>
+  
+  <variablelist>
+   <varlistentry><term>Structure the table to have more rows than columns</term>
+    <listitem>
+     <para>Readers prefer to skim for information that is aligned vertically. When designing 
+      tables, consider swapping rows and columns to provide a consistent
+      user experience.</para>
+    </listitem>
+   </varlistentry>
+   <varlistentry><term>Narrow down columns</term>
+    <listitem>
+     <para>Compress the data in your table as much as you possibly can. Table
+      cells with less content are more easily parsed by readers. Consider using
+      icons, adding repeating words into column titles, or using shorter number 
+      formats.</para>
+    </listitem>
+   </varlistentry>
+   <varlistentry><term>Use colors to lead the reader's eyes</term>
+    <listitem>
+     <para>Use colors to make the information stand out. If you just want to
+      highlight small bits, use bright colors. If you need to color entire cells, 
+      use pastels. This option should only be used with matrix type tables.</para>
+     <!-- add list of supported colors as soon as toms' PR is merged -->
+    </listitem>
+   </varlistentry>
+   <varlistentry><term>Use striped rows</term>
+    <listitem>
+     <para>Color every second row in light gray to make sure readers do not 
+      accidentally slip between lines. This should be the default for long 
+      tables. Do not use striped rows in matrix type tables. You may also consider disabling 
+      these stripes for short tables to not confuse readers.</para>
+     <!-- find out whether there is an easy way to do this instead of coloring each and every cell -->
```

Comment:
I'm not sure if we should mention that explicitly. This is usually done by the (CSS) stylesheet automatically.

Reply:
I'd still leave it in to provide some sort of rationale why we are using the stripes and when they don't add value.